### PR TITLE
ERSSUP-57401 Updating overview and examples for change shortlist

### PR DESCRIPTION
**Pull request by francisco-videira-nhs**
## Summary
* Routine Change

1. Added Overview and examples to the change shortlist specification.

2. Included headers and content for 422 response.

3. Tested sending requests to the sandbox on the dev-vm using Postman and the appropriate/corresponding responses are sent back.

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: sandbox/src/mocks/changeShortlist/responses/UnbookedReferralMultipleServices.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
+{
```
Comment (nhsd-jack-wainwright) [inline sandbox/src/mocks/changeShortlist/responses/UnbookedReferralMultipleServices.json:1]: I think this response should also be included as an example for a successful 200 response in the OAS documentation.

**Review by nhsd-jack-wainwright — APPROVED** [review]
Looks good to me 🙂, just a minor comment on including the new example in the OAS documentation as well as it being accessible via the sandbox.

File: sandbox/src/mocks/changeShortlist/responses/UnbookedReferralMultipleServices.json (hunk 1 above)
Comment (francisco-videira-nhs) [reply sandbox/src/mocks/changeShortlist/responses/UnbookedReferralMultipleServices.json:1]: Good spot! I've added that now

**Pull request merged**